libraries/ESP8266WiFi/src/ESP8266WiFiMulti.cpp

```diff
@@ -38,8 +38,104 @@ bool ESP8266WiFiMulti::addAP(const char* ssid, const char *passphrase) {
     return APlistAdd(ssid, passphrase);
 }
 
+void ESP8266WiFiMulti::handleConnectComplete(wl_status_t status) {

```

Comment:
This function has no dependencies on other class members or methods, and is not public. Please make it a static free standing function.